services.yaml

```diff
@@ -2045,12 +2051,12 @@ paths:
     post:
       tags:
         - Labels
-      summary: "Create new label"
-      description: "Create new label"
+      summary: "Define new label"
```

Comment:
I think that 'create' is better than 'define'. 'Define' doesn't feel like creating something new and sounds more as 'customize existing'

Reply:
i agree